File: fizzbuzz_js/src/fizzBuzz.js
class FizzBuzz {
  check(value) {
    if (value % 3 === 0) {
      return "fizz";
    } else if (value % 5 === 0) {
      return "buzz";
    } else {
      return value;
    }
  }
}

module.exports = FizzBuzz;

File: fizzbuzz_rb/src/fizz_buzz.rb
class FizzBuzz
  def check(value)
    if value % 15 == 0
      'fizz-buzz'
    elsif value % 3 == 0
      'fizz'
    elsif value % 5 == 0
      'buzz'
    else
      value
    end
  end
end
